tests/unistd/unlink.c

```diff
@@ -52,10 +52,14 @@ void setup() {
 #ifndef NO_SYMLINK
   symlink("dir-empty", "dir-empty-link");
 #endif
+#ifndef WASMFS
```

Comment:
It would be good to add a comment explaining why there is a difference here and when we will be able to unify the code, if ever.

Reply:
Yep, it's because `chmod` is not implemented yet.